### ISSUE TITLE
how to custom a ELBO with weighted binary cross entropy loss in VAE

### ISSUE THREAD

**Issue by AmyTanJ** [pyro-ppl/pyro#2973]
The elbo in PyTorch is divided into two parts, KL and reconstruction loss. My current dataset is a group of sparse binary matrices containing a small number of 1value and most of the value. I want to use the weighted binary cross-entropy in the reconstruction loss and give a higher weight to the 1（Just like the formula in the picture, weight 1 value beta> 1 ）for better results in reconstruction, but I don’t know how to customize the elbo in pyro to achieve my goal.
![image](https://user-images.githubusercontent.com/73063292/144353005-130fd642-f36f-4bc0-a8e7-7b9c883d3ce8.png)

**Comment by martinjankowiak** [comment]
closing this in favor of the [forum post](https://forum.pyro.ai/t/about-custom-elbo/3610/2), which is the preferred venue for these kinds of questions
